cpu/kinetis/periph/pwm.c

```diff
@@ -21,29 +21,50 @@
  * @author      Johann Fischer <j.fischer@phytec.de>
  * @author      Jonas Remmert <j.remmert@phytec.de>
  * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ * @author      Thomas Stilwell <stilwellt@openlabs.co>
  *
  * @}
  */
 
 #include "cpu.h"
 #include "assert.h"
 #include "periph/pwm.h"
+#include "pm_layered.h"
 
-/* This driver uses the FlexTimer module (FTM) for generating PWM signals, but
- * not all Kinetis CPUs have such a module. This preprocessor condition prevents
- * compilation errors when the CPU header lacks the register definitions for the
- * FTM */
-#ifdef FTM0
+/* This mode will be blocked while PWM is active */
+#ifndef PWM_PM_BLOCKER
+#define PWM_PM_BLOCKER           KINETIS_PM_STOP
+#endif
+
+/**
+ * @brief   Keeps track of whether PWM is active for power management
+ */
+static bool _is_powered_on[PWM_NUMOF];
```

Comment:
bool is a bit wasteful.
How about a bitmap

```C
static uint32_t _is_powered_on;
```

if there can be no more than 32 pwms.